e2e/setup.go

```diff
@@ -123,7 +125,7 @@ func DefaultSystemConfig(t *testing.T) SystemConfig {
 		ColosseumSegmentsLengths:  "3,3",
 
 		SecurityCouncilNumConfirmationRequired: 1,
-		SecurityCouncilOwners:                  []common.Address{addresses.Alice, addresses.Bob, addresses.Mallory},
+		SecurityCouncilOwners:                  []common.Address{addresses.Challenger, addresses.Alice, addresses.Bob, addresses.Mallory},
```

Comment:
Why is the challenger a member of the security council here?

Reply:
I am simulating a situation where the challenger node and the guardian are activated together. To do this, I added challenger to security council owners.

Reply:
If the challenger sends challenges in tests, the challenger should not be security council.